modules/pulsepointBidAdapter.js

```diff
@@ -58,13 +58,6 @@ export const spec = {
         url: 'https://bh.contextweb.com/visitormatch/prebid'
       }];
     }
-  },
-  transformBidParams: function(params) {
-    return convertTypes({
```

Comment:
you need to stop importing this on line 4 above to pass the linting check

Reply:
addressed